### ISSUE TITLE
Please let us copy the simple verse text

### ISSUE THREAD

**Issue by Nefcanto** [quran/quran_android#2369]
Right now we can long-hold a verse, choose sharing and copy it.

But the copied text includes curly braces, brakets, and chapter info.

Please give us another option to only copy the text verse as is, without any extra characters.

Thank you

**Comment by MahmoudMabrok** [comment]
is there a case we need to copy only simple text? @Nefcanto

**Comment by Nefcanto** [comment]
@MahmoudMabrok, yeah, for example, I'm learning Arabic and memorizing the Quran. And I want to copy/paste verse texts without verse numbers and chapter numbers into the AnkiDroid (a flashcard app).

Or I want to send a friend of mine a verse showing him what the Quran says about a specific topic.

**Comment by MahmoudMabrok** [comment]
I will give it a try and will try you what the result I reach.

**Comment by ahmedre** [comment]
I think we need to rethink the sharing experience in general at some point to have this all be via a dialog allowing setting some of the options - that having been said I am not sure we should do this considering it seems like a very specific case, though I think something like a "minimal" version (i.e. 1:1 at the end of the ayah instead of [Sura Fatiha: Ayah 1]) might make sense when we do this type of sharing experience.

**Comment by MahmoudMabrok** [comment]
totally agree @ahmedre, is there any open requests for  sharing experience, or any link to an initial re-vamp so we can collect all and start planning.

**Comment by Nefcanto** [comment]
Quran is a holy book and we all respect it. But I disagree with the necessity of accompanying it with metadata.

Even many times while we speak with our friends we refer to it because we have memorized some major lessons of it, but we don't know which Surah and Ayah it was.

For example, I tell my friend:

> Dont do it, Quran says "و لا تقربوا الزنا"

What could be wrong about it? 

The same holds in chats and everyday conversations. We shouldn't make it hard for the community to use the Quran and share parts of it.

**Comment by ahmedre** [comment]
@Nefcanto you misunderstand me - I am not saying it's necessary or not, and I think your use case is valid. However, as developers, adding a feature means you have to maintain it forever, and so a balance needs to be struck between adding features. Too many features that only very few people use make it more difficult to maintain the code and it's really hard to get rid of features after you add them (examples today in the app - the "old background," and the "volume buttons to change pages."). This is the first time to hear this request, which makes me think it's not a very common use case.

**Comment by ahmedre** [comment]
@MahmoudMabrok not sure yet, but some thoughts:

1. should gather all the sharing types (copy, link, text) into that.
2. should have a "share as an image" option (people share text today to devices without uthmani fonts and so sometimes see things they do not expect).
3. should perhaps handle #942 (perhaps allowing selection of a translation or none, etc)
4. should perhaps handle #2072 (perhaps allowing handling of sharing audio)

**Comment by iMJ007** [comment]
I would like to put forth my suggestion of sharing ayah and translation in non-editable form such as cards/images as translation texts are sometimes edited in a misleading way by miscreants.
